src/pages/instances/actions/DeleteInstanceBtn.tsx

```diff
@@ -42,11 +44,11 @@ const DeleteInstanceBtn: FC<Props> = ({ instance }) => {
   return (
     <ConfirmationButton
       onHoverText="Delete instance"
-      toggleAppearance="base"
+      toggleAppearance={isSmallScreen ? "base" : "default"}
       className="delete-instance-btn"
```

Comment:
Please remove this custom class and it's styling in the scss file. We should use u-no-margin--bottom instead.